frontend/src/styles.css

```
@@ -4181,3 +4181,839 @@ label {
+
+.goal-detail-card {
+  display: grid;
+  grid-template-columns: minmax(210px, 0.33fr) minmax(0, 1fr);
+  gap: 1.5rem;
+  align-items: center;
+  padding: clamp(1.2rem, 3vw, 2.1rem);
+}
+
+.goal-ring {
+  --goal-progress: 0%;
+  display: grid;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)